templates/monitoring/kube_state_metrics_3scale_alerts.yaml

```diff
@@ -94,6 +94,24 @@ spec:
         for: 5m
         labels:
           severity: critical
+      - alert: ThreeScaleZyncPodAvailability
+        annotations:
+          message:  3Scale Zync has no pods in a ready state.
+          sop_url: https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md
+        expr: |
+          absent(count((kube_pod_status_ready{condition="true",namespace="redhat-rhmi-3scale",pod=~"zync-1.*"}) + on(pod) (kube_pod_container_status_running{namespace="redhat-rhmi-3scale",container="zync"}))==2)
```

Comment:
as discussed in person, this may start triggering if a new deploy is kicked off (incrementing the deploy number).
I'd suggest matching based on pod labels (by joining with kube_pod_labels_info metric)

Reply:
agree, changed the rule to check for label
```
((kube_pod_labels{ namespace="redhat-rhmi-3scale", pod=~"zync-database.*",label_threescale_component_element="database"})+on(pod)(kube_pod_status_ready{ namespace="redhat-rhmi-3scale", pod=~"zync-database.*",condition="true" })!=2)```